### ISSUE TITLE
Does this works for multi-person action recognition?

### ISSUE THREAD

**Issue by athulvingt** [murilovarges/HARBoP#1]

**Comment by murilovarges** [comment]
Yes, I'm working on it just now. Basically, you need to perform the tracking of each person and then apply the proposed method.

**Comment by athulvingt** [comment]
I shall be obliged if you could explain me how to feed in a video and run this, or attach a code to run this on a video

**Comment by athulvingt** [comment]
do u have code snippet to run it on a video?

I guess using this tensorflow implementation of  openpose https://github.com/ildoonet/tf-pose-estimation would be easier to run it in realtime,
